docs/docs/stories.mdx

```diff
@@ -32,6 +33,9 @@ stories:
   - action: utter_ask_num_people
 ```
 
+See the [training data format](./training-data-format.mdx#stories)
+for detailed description of how to write stories.
```

Comment:
🤔 why?

Reply:
I was asked in one of the comments above. I can remove that